lib/rules/rule-empty-line-before/index.js

```diff
@@ -96,7 +97,8 @@ const rule = function(expectation, options, context) {
           isNested &&
           isAfterRule(rule)) ||
         (optionsMatches(options, "except", "after-single-line-comment") &&
-          isAfterSingleLineComment(rule))
+          isAfterSingleLineComment(rule)) ||
+        (optionsMatches(options, "except", "inside-block") && isNested)
```

Comment:
Just move isNested to a new line to be constistent with the other conditions.

Reply:
@CAYdenberg If I change it prettier fixes this back. I suppose this line is too short to break it.

Reply:
Hm. Ok then. More of a personal preference that anything else.